uavcan/tunnel/2011.SerialConfig.uavcan

```diff
@@ -0,0 +1,5 @@
+#
+# This message carries baudrate for tunnel channel_id
+
+uint8 channel_id
```

Comment:
I'm not sure how this works or what this means. Some description might be helpful.

Reply:
It's still not clear to me what this means or how it works, but I'm probably missing UAVCAN context.